app/controllers/communities_controller.rb

```diff
@@ -0,0 +1,52 @@
+class CommunitiesController < ApplicationController
+  before_action :require_community
+  before_action :authenticate_user!, only: [:new, :edit]
+  skip_before_action :require_community, only: [:new, :create]
+
+  def show # First add association to Posts
```

Comment:
šis komentars nenes logisko slodzi, vari viņu izņemt :D

Reply:
oh , es to biju pielicis pasa sakuma lai neaizmirstu pievienot association :D , aizmirsas iznemt